Sources/Composed/Sections/SingleElementSection.swift

```diff
@@ -26,9 +26,10 @@ open class SingleElementSection<Element>: Section {
     public init(element: Element) {
         self.element = element
 
-        switch element as Any {
-        case Optional<Any>.none: numberOfElements = 0
-        default: numberOfElements = 1
+        if case Optional<Any>.none = element as Any {
```

Comment:
rather than have identical business logic in two places, perhaps a private `_replace(element:)` method that ommits the delegate calls and `diffing` then call it from both places?

Reply:
I'll open a separate PR for this (this PR should eventually have no difference, but it's useful as a checklist of things that need PRs creating)